apps/mobile-wallet/src/screens/PublicKeysScreen.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { StackScreenProps } from '@react-navigation/stack'
+import * as Clipboard from 'expo-clipboard'
+
+import { sendAnalytics } from '~/analytics'
+import AddressBadge from '~/components/AddressBadge'
+import { BackButton } from '~/components/buttons/Button'
+import FlatListScreen from '~/components/layout/FlatListScreen'
+import { ScrollScreenProps } from '~/components/layout/ScrollScreen'
+import Row from '~/components/Row'
+import { useAppSelector } from '~/hooks/redux'
+import RootStackParamList from '~/navigation/rootStackRoutes'
+import { selectAllAddresses } from '~/store/addressesSlice'
+import { showToast, ToastDuration } from '~/utils/layout'
+
+interface PublicKeysScreenProps extends StackScreenProps<RootStackParamList, 'PublicKeysScreen'>, ScrollScreenProps {}
```

Comment:
It should extend `FlatListScreenProps` (see `AddressFlatListScreenProps` as an example)

Reply:
What I did initially, but then the `RootStackNavigation` complained.
I realise now that the `AddressFlatListScreen` is always used inside a Screen declared as a `ScrollScreen` (cf. `OriginScreen`, `AddressScreen`) which is why the compiler isn't screaming. But it's wrong too..

Reply:
If you don't want to figure out how we should properly fix this in all our screens, please create an issue on GH and reference it in a TODO 🙏